lib/fpm/util.rb

```diff
@@ -298,8 +298,8 @@ def copy_entry(src, dst, preserve=false, remove_destination=false)
       if known_entry
         FileUtils.ln(known_entry, dst)
       else
-        FileUtils.copy_entry(src, dst, preserve=preserve,
-                             remove_destination=remove_destination)
+        FileUtils.copy_entry(src, dst, preserve, false,
```

Comment:
Good catch! 

I have looked around fpm and the ruby docs and I'm not really sure what some of these arguments to FileUtils::copy_entry really mean.